pkg/deployment/reconcile/helper_shutdown.go

```diff
@@ -75,10 +77,17 @@ func (s shutdownHelperAPI) Start(ctx context.Context) (bool, error) {
 			return false, errors.WithStack(err)
 		}
 		removeFromCluster := false
-		log.Debug().Bool("removeFromCluster", removeFromCluster).Msg("Shutting down member")
+		gracefulShutdown := false
+		if isGracefulShutdownAvailable(group, m.ArangoVersion) {
```

Comment:
Is this needed? Anyway this param should be ignored on older versions.

Reply:
It is required because a graceful shutdown is available from the specific version and only it applies to the coordinator. Everything is checked in the function `isGracefulShutdownAvailable`

Reply:
removed